### ISSUE TITLE
control-F toggle lacks strong visual indicator 

### ISSUE THREAD

**Issue by reduxdj** [zed-industries/feedback#670]
### Check for existing issues

- [X] Completed

### Is your feature request related to a problem?

UI doesn't reflect enough change when I hit `control-f`, so I miss it.

### Describe the solution you'd like

I'd like the control-F toggle to make the UI a little more apparent that control-f is enabled.

**Comment by JosephTLyons** [comment]
Hey @reduxdj, are you referring to `cmd-f`, for searching within the current file?  `control-f` advances the cursor one character at a time for me.

**Comment by JosephTLyons** [comment]
Just checking back in.  I'm thinking that maybe this is referring to the highlighting of the matched words being too subtle?

**Comment by reduxdj** [comment]
Yes, sorry for not responding sooner,  and you are right, i do mean command-f, my bad. Yes, also of course i've been using Atom, so maybe I didn't notice the placement (at top rather than bottom), but if the border stroke on the text input and/or highlight wasn't so subtle, it would be easier to notice this one.


<img width="927" alt="Screen Shot 2022-11-29 at 7 51 52 PM" src="https://user-images.githubusercontent.com/322547/204703584-3f3869f7-d615-44ef-9bae-b6fde15fe8fd.png">
